snuba/datasets/configuration/generic_metrics/storages/sets.yaml

```diff
@@ -0,0 +1,53 @@
+version: v1
+kind: readonly_storage
```

Comment:
I think we should call this a readable storage (not `readonly`). For generic metrics, the writable/readable are separate but for many entities/datasets these things aren't separate